### PR TITLE
Add recipe for python-ldap

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

File: recipes/python-ldap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - pyasn1-modules
+    - openldap
+
+test:
+  imports:
+    - ldap
+    - ldap.controls
+    - ldap.extop
+    - ldap.schema
+    # - slapdtest
```
Comment (beckermr) [inline recipes/python-ldap/meta.yaml]: Do you need this comment or shall we remove it?

File: recipes/python-ldap/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - pyasn1
+    - pyasn1-modules
+    - openldap
```
Comment (beckermr) [inline recipes/python-ldap/meta.yaml]: Are these deps needed for installing or just running?

Comment (mariusvniekerk) [reply recipes/python-ldap/meta.yaml]: They're needed to compile the c extension

File: recipes/python-ldap/meta.yaml (hunk 1 above)
Comment (mariusvniekerk) [reply recipes/python-ldap/meta.yaml]: It we decide to add slapd to openldap it may be handy to uncomment

**Pull request merged**